package.json

```diff
@@ -502,6 +502,34 @@
           "default": "file=*",
           "description": "Default filter criteria for Local Files",
           "scope": "window"
+        },
+        "zowe.cics.allPrograms.recordCountIncrement": {
+          "type": "number",
+          "default": 500,
+          "description": "Default filter criteria for all Programs",
+          "scope": "window",
+          "minimum": 1
+        },
+        "zowe.cics.allTransactions.recordCountIncrement": {
+          "type": "number",
+          "default": 500,
+          "description": "Default filter criteria for all Transactions",
+          "scope": "window",
+          "minimum": 1
+        },
+        "zowe.cics.allLocalFiles.recordCountIncrement": {
+          "type": "number",
+          "default": 500,
+          "description": "Default filter criteria for all Local Files",
```

Comment:
It might be better to change the description to "Default record count increment for 'All Local Files'"